cpp/doxygen/Doxyfile

```diff
@@ -287,7 +287,7 @@ ALIASES                =
 # members will be omitted, etc.
 # The default value is: NO.
 
-OPTIMIZE_OUTPUT_FOR_C  = NO
+OPTIMIZE_OUTPUT_FOR_C  = YES
```

Comment:
This looks odd. Ice C++ is not a C project.

Reply:
You are right, this is only for C, not C++